_data-prepper/core-apis.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: Core APIs
+nav_order: 2
+---
+
+# Core APIs
+
+All Data Prepper instances expose a server with some control APIs. By default, this server runs on port 4900. Some plugins, especially Source plugins may expose other servers. These will be on different ports and their configurations are independent of the core API. For example, to shut down Data Prepper, you can run:
+
+```
+curl -X POST http://localhost:4900/shutdown
+```
+
+## APIs
+
+The following APIs are available:
```

Comment:
```suggestion
The following table lists the available APIs.
```

Reply:
@natebower Is there a reason we aren't supposed to use a colon after APIs here?

Reply:
@carolxob When text introduces a table or image, it should be a complete sentence and end with a period, not a colon.